app/[productId]/page.tsx

```
@@ -0,0 +1,69 @@
+        } finally {
+            setLoading(false)
+        }
+    }
+
+    const addToCart = (selectedProductId: string) => {
+        setLoading(true)
+        try {
+            
+        } catch (error) {
+            console.log(error)
+        } finally {
```

Biome (1.9.4)

[error] This code is unreachable



(lint/correctness/noUnreachable)